examples/platforms/cc2538/random.c

```diff
@@ -78,7 +106,7 @@ ThreadError otPlatSecureRandomGet(uint16_t aInputLength, uint8_t *aOutput, uint1
 
     for (uint16_t length = 0; length < aInputLength; length++)
     {
-        aOutput[length] = (uint8_t)otPlatRandomGet();
+        aOutput[length] = (uint8_t)generateRandom();
```

Comment:
Might be more efficient to just have `generateRandom()` take a buffer and size as arguments.  Otherwise you will have to keep enabling/disabling the radio for each byte.